### DISCUSSION TITLE
NetBox sync with Oracle Cloud infrastructure

### DISCUSSION THREAD

**Discussion by AnuragYewale15** [bb-Ricardo/netbox-sync#380] [General]
Hi @bb-Ricardo ,
I was wondering if there is any way to integrate the Oracle Cloud Infrastructure(OCI) with NetBox in order to get data related to VMs from OCI and to display it in Netbox. Do we need any changes in your code or we can do it from existing sync as it say from different sources. Your help would be appreciated.
Thanks!

**Comment by bb-Ricardo** [comment]
Hi,

Oracle Cloud Infrastructure is not supported. Someone would need to implement this source.
